src/jsglue.cpp

```diff
@@ -1077,6 +1078,16 @@ JS_GetPromiseResult(JS::HandleObject promise, JS::MutableHandleValue dest) {
   dest.set(JS::GetPromiseResult(promise));
 }
 
+void
+JS_GetScriptPrivate(JSScript* script, JS::Value* dest) {
```

Comment:
We should use `JS::MutableHandleValue dest` like JS_GetPromiseResult (and that will match what's declared in glue.rs as well).

Reply:
> and that will match what's declared in glue.rs as well

Indeed! Thanks for catching this! Updated!